docs/content/blog/using-docker-in-bundles.md

```diff
@@ -77,138 +76,100 @@ required:
 
 ### Install Docker
 
-We can install the Docker CLI, or whatever is needed, into the
-bundle using a [custom Dockerfile](https://porter.sh/custom-dockerfile/).
+You can include the Docker CLI in your bundle by using the [docker mixin], or
+if you are using Docker Compose, use the [docker-compose mixin].
 
-In our [Dockerfile.tmpl](https://porter.sh/src/examples/docker/Dockerfile.tmpl)
-below, I am installing the Docker CLI and copying my files into the bundle:
-
-```dockerfile
-FROM debian:stretch-slim
-
-ARG BUNDLE_DIR
-RUN apt-get update && apt-get install -y curl ca-certificates
-
-ARG DOCKER_VERSION=19.03.8
-RUN curl -o docker.tgz https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_VERSION}.tgz && \
-    tar -xvf docker.tgz && \
-    mv docker/docker /usr/bin/docker && \
-    chmod +x /usr/bin/docker && \
-    rm docker.tgz
-
-# Use the BUNDLE_DIR build argument to copy files into the bundle
-COPY . $BUNDLE_DIR
+```yaml
+mixins:
+  - docker
+  - docker-compose
 ```
 
-### The docker-compose mixin
-
-Alternatively, we have also created a new mixin,
-[docker-compose](https://porter.sh/mixins/docker-compose/), that helps you
-quickly use Docker Compose from inside your bundle. It handles installing the
-Docker Compose CLI and gives you the following syntax:
+You can then use the respective CLIs through the mixin:
 
 ```yaml
-docker-compose:
-  description: "Start Test Services"
-  arguments:
-  - up
-  - -d
+install:
+- docker:
+    description: "Run my container"
+    run:
+      image: hello-world
+      rm: true
+- docker-compose:
+    description: "Start Test Services"
+    arguments:
+    - up
+    - -d
 ```
 
 We are going to focus on just using Docker for this blog post, but here is a [full
```

Comment:
Shift in pronouns here, changing it to I would match up the following example (instead of We)